src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/table.html.twig

```diff
@@ -55,4 +63,12 @@
   </tbody>
   {% block grid_table_footer %}{% endblock %}
 </table>
+
+{{ renderhook('displayAdminGridTableAfter', {
```

Comment:
Should be outside the form. What if you want to include your own form inside it? :thinking: